src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -2,20 +2,41 @@
 
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 public class DatabaseManager {
     private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
 
     private Connection connection;
 
-    boolean connect(String database, String user, String password) {
+    public boolean connect(String database, String user, String password) {
         try {
             Class.forName("org.postgresql.Driver");
-            connection = DriverManager.getConnection(DB_CONNECTION_URL + database, user, password);
+            connection = DriverManager.getConnection(DB_CONNECTION_URL + database+"?loggerLevel=OFF", user, password);
             return true;
         } catch (ClassNotFoundException | SQLException e) {
             return false;
         }
     }
+
+    public String[] getTableNames() throws SQLException {
+        String sqlQuery = "SELECT relname FROM pg_stat_user_tables";
+        try (Statement statement = connection
```

Comment:
Why second line?

Reply:
I was trying to go according to requirement from Google Java Style Guide - http://bit.ly/2vljPH6. I also don't like this place ) Is it ok to use here single line and have 126 characters in one line? Probably you can recommend better line-wrapping in this case. Thanks!